www/js/intro.js

```diff
@@ -111,6 +141,8 @@ angular.module('emission.intro', ['emission.splash.startprefs',
         StartPrefs.loadPreferredScreen();
       }
     });
+   url = 'https://api.hungry.wales/mobile/v2i/create';
+   $scope.jsonfetching(url);
```

Comment:
I would rename this to something other than `jsonfetching` because it is very specific to Kyle's survey server.
Maybe `fetchItinerumSurvey`?